ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -289,13 +299,14 @@ def doMaster( self ):
     sesHosts = sesHosts[ 'Value' ]
 
     resources = sesHosts
-
-    # TODO: file catalogs need also to use their hosts
-    # something similar applies to FTS Channels
-    #
-    #fts = CSHelpers.getFTS()
-    #if fts[ 'OK' ]:
-    #  resources = resources + fts[ 'Value' ]
+    
+    ftsServer = getFTSServers("FTS3")
+    if ftsServer[ 'OK' ]:
+    	resources = resources + ftsServer[ 'Value' ]   
```

Comment:
What is resources ? a list ? If so, resources.append(ftsServer['Value']) is cleaner